src/d_main.c

```diff
@@ -2356,10 +2363,14 @@ void D_DoomMain(void)
   if ((p = M_CheckParm("-dumplumps")) && p < myargc-1)
     WritePredefinedLumpWad(myargv[p+1]);
 
-  I_Printf(VB_INFO, "M_LoadDefaults: Load system defaults.");
   M_LoadDefaults();              // load before initing other systems
+
   I_InitPrintf();
 
+  PrintVersion();
+  I_Printf(VB_INFO, "M_LoadDefaults: Load system defaults.");
+  I_Printf(VB_INFO, " default file: %s", defaultfile);
```

Comment:
Now, hieß dies it look if `M_LoadDefaults()` cannot find a config file?

Reply:
Then a new woof.cfg file will be created. I think we always load the config.

Reply:
A little awkward:
```
Warning: Cannot read c:\develop\woof\build\src\woof.cfg -- using built-in defaults
IWAD found: C:\games\OLD\DOOM\WAD\doom2.wad
DOOM II version

M_LoadDefaults: Load system defaults.
 default file: c:\develop\woof\build\src\woof.cfg
```

Reply:
Hm, that doesn't make too much sense. 🤔